src/test/java/org/folio/circulation/domain/policy/PeriodTest.java

```diff
@@ -39,4 +47,30 @@ public void toMinutesWithUnknownInterval() {
     Period period = Period.from(10, "Unknown interval");
     assertEquals(0, period.toMinutes());
   }
-}
\ No newline at end of file
+
+  @Test
+  public void periodsWithDifferentIntervalNamesIsNotEqual() {
+    val oneMonth = months(1);
+    val thirtyOneDay = days(31);
```

Comment:
If this test is intended to demonstrate the check for the interval only, then the duration needs to be the same for both example periods, otherwise this will succeed because of only a comparison of the duration.

Reply:
This test to verify that 1 month is not equal to 4 weeks or 31 days. Two period should be considered as equal if both interval name and duration are the same.

Reply:
Hmm, ok, then I think we need specific names for those e.g. `oneMonthShouldNotEqualFourWeeks` and `oneMonthShouldNotEqual31Days`

I think that is separate to testing whether periods with different intervals in general are not equal.

Does that make sense?

Reply:
Yes, thanks